evento/app/controllers/users_controller.rb

```diff
@@ -63,7 +64,11 @@ def set_user
   end
 
   # Only allow a trusted parameter "white list" through.
-  def user_params
-    params.require(:user).permit(:name, :email, :password)
+  def create_params
```

Comment:
Oddly enough, in tests, if line

`put "/users/#{user.id}", params: { name: "NewName" }, headers: auth_header`

is replaced with

`put "/users/#{user.id}", params: { "name": "NewName"}.to_json, headers: { "Authorization": json["auth_token"], 'CONTENT_TYPE' => 'application/json' }`

the tests pass.